driver-benchmarks/build.gradle

```diff
@@ -30,14 +30,17 @@ sourceSets {
 }
 
 dependencies {
+    compile project(':bson')
     compile project(':driver-sync')
     compile 'ch.qos.logback:logback-classic:1.1.1'
 }
 
 javadoc {
     enabled = false
 }
+jar {
```

Comment:
We will no longer publish a JAR file for `driver-benchmarks`.

Reply:
Is this needed, since we have this in publish.gradle:

    def javaProjects = subprojects.findAll { !scalaProjects.contains(it) && !['util', 'driver-benchmarks'].contains(it.name) }

Reply:
Ah this is because we apply the bnd plugin in the root `build.gradle`  and not in `publish.gradle` that means it runs on all Java projects and not just the ones we publish.  Try moving the plugin into publish.gradle and see if that fixes it.

Reply:
I have moved the bnd apply to `publish.gradle`, but a JAR file still gets generated for `driver-benchmarks`. The difference now is that the generated manifest only has one line:
```
Manifest-Version: 1.0
```